wormhole-connect/src/utils/wallet/index.ts

```diff
@@ -160,6 +157,8 @@ export const postVaa = async (
   if (!wallet) throw new Error('not connected');
   const pk = (wallet as any).adapter.publicKey;
   const MAX_VAA_UPLOAD_RETRIES_SOLANA = 5;
+  const computeBudgetInstructions =
+    await solanaContext().determineComputeBudget([]);
```

Comment:
It's difficult to know which accounts to pass here since we have to know the route from which this VAA is for. Even then it seems like it would be complex and a lot of code to figure that out. I think it's sufficient for the fee to be at least 100_000 micro-lamports here.